src/test/kotlin/io/github/mojira/arisa/utils/MockDomain.kt

```diff
@@ -105,6 +105,7 @@ fun mockIssue(
     updateDescription: (description: String) -> Unit = { Unit },
     updateCHK: () -> Unit = { Unit },
     updateConfirmationStatus: (String) -> Unit = { Unit },
+    updatePriority: (String) -> Unit = { Unit },
```

Comment:
This should also be referenced in the body's constructor

Reply:
thanks, i just saw that :)